modules/light-clients/08-wasm/testing/wasm_endpoint.go

```diff
@@ -15,7 +17,8 @@ type WasmEndpoint struct {
 }
 
 var (
-	CodeHash               = []byte("01234567012345670123456701234567")
+	// Represents the code of the wasm contract used in the tests with a mock vm.
+	Code                   = []byte("01234567012345670123456701234567")
```

Comment:
I think its best to use this as the dummy wasm code? If we use it as the codeHash we wouldn't be able to re-use it in any spot where we might need to test/invoke `StoreCode`.